tests/admin_inlines/tests.py

```diff
@@ -1853,7 +1853,7 @@ def test_delete_inlines(self):
     def test_collapsed_inlines(self):
```

Comment:
I think it's worth us adding in some screenshots to these tests. Something like:
```diff
diff --git a/tests/admin_inlines/tests.py b/tests/admin_inlines/tests.py
index 91a8affba5..40db1dd040 100644
--- a/tests/admin_inlines/tests.py
+++ b/tests/admin_inlines/tests.py
@@ -4,7 +4,9 @@ from django.contrib.admin.tests import AdminSeleniumTestCase
 from django.contrib.auth.models import Permission, User
 from django.contrib.contenttypes.models import ContentType
 from django.test import RequestFactory, TestCase, override_settings
+from django.test.selenium import screenshot_cases
 from django.urls import reverse
+from django.utils.translation import gettext

 from .admin import InnerInline
 from .admin import site as admin_site
@@ -1850,6 +1852,7 @@ class SeleniumTests(AdminSeleniumTestCase):
             "form#profilecollection_form tr.dynamic-profile_set#profile_set-2", 1
         )

+    @screenshot_cases(["desktop_size", "mobile_size", "rtl", "dark"])
     def test_collapsed_inlines(self):
         from selenium.webdriver.common.by import By

@@ -1865,15 +1868,19 @@ class SeleniumTests(AdminSeleniumTestCase):
         ]
         summaries = self.selenium.find_elements(By.TAG_NAME, "summary")
         self.assertEqual(len(summaries), 3)
+        self.take_screenshot("loaded")
         for show_index, field_name in enumerate(test_fields, 0):
             self.wait_until_invisible(field_name)
             summaries[show_index].click()
             self.wait_until_visible(field_name)
+        self.take_screenshot("expanded")
         for hide_index, field_name in enumerate(test_fields, 0):
             self.wait_until_visible(field_name)
             summaries[hide_index].click()
             self.wait_until_invisible(field_name)
+        self.take_screenshot("collapsed")

+    @screenshot_cases(["desktop_size", "mobile_size", "rtl", "dark"])
     def test_added_stacked_inline_with_collapsed_fields(self):
         from selenium.webdriver.common.by import By

@@ -1881,18 +1888,22 @@ class SeleniumTests(AdminSeleniumTestCase):
         self.selenium.get(
             self.live_server_url + reverse("admin:admin_inlines_teacher_add")
         )
-        self.selenium.find_element(By.LINK_TEXT, "Add another Child").click()
+        add_text = gettext("Add another %(verbose_name)s") % {"verbose_name": "Child"}
+        self.selenium.find_element(By.LINK_TEXT, add_text).click()
         test_fields = ["#id_child_set-0-name", "#id_child_set-1-name"]
         summaries = self.selenium.find_elements(By.TAG_NAME, "summary")
         self.assertEqual(len(summaries), 3)
+        self.take_screenshot("loaded")
         for show_index, field_name in enumerate(test_fields, 0):
             self.wait_until_invisible(field_name)
             summaries[show_index].click()
             self.wait_until_visible(field_name)
+        self.take_screenshot("expanded")
         for hide_index, field_name in enumerate(test_fields, 0):
             self.wait_until_visible(field_name)
             summaries[hide_index].click()
             self.wait_until_invisible(field_name)
+        self.take_screenshot("collapsed")

     def assertBorder(self, element, border):
         width, style, color = border.split(" ")
```

This gives me a couple more screenshots that I think we might need to do some css tweaking for, specifically...
"Name" is on the left for RTL and generally the contrast of this looks not strong enough to me
![image](https://github.com/django/django/assets/42296566/e44ee7db-99bd-43b3-9d14-e89163747876)

I haven't investigated if these are existing issues yet 🤔 might well be

Reply:
@sarahboyce I've added these lines of code, see the artifacts for screenshots.

Reply:
@MHLut I have some slightly bad news, the color contrast is currently an issue on this patch because the css was previously redundant and never applied. With the new html, it has resurrected this dodgy css 

Previously the header would look like this:
![image](https://github.com/django/django/assets/42296566/8fdf2777-3353-4c29-8d8f-9b1a36136a2f)

There is a ticket to delete this css here: https://code.djangoproject.com/ticket/35382
This needs to happen before this PR can be merged

Reply:
@sarahboyce Well, I dislike this light blue heading since it doesn't match the modern admin style. I'm on board with making it look more like the current style on `main` again.

See also the comment I placed on that ticket.

Either someone removes the redundant CSS in a separate PR, or I'll remove it in this PR; it doesn't matter because I will have to update the styles anyway.